workflow_trial/pipeline.py

```diff
@@ -0,0 +1,29 @@
+import datajoint as dj
+
+from element_lab import lab
+from element_animal import subject
+from element_session import session
+
+from element_animal.subject import Subject
+from element_lab.lab import Source, Lab, Protocol, User, Project
+from element_session.session import Session
+
+if 'custom' not in dj.config:
+    dj.config['custom'] = {}
+
+db_prefix = dj.config['custom'].get('database.prefix', '')
+
+
+# Activate "lab", "subject", "session" schema -------------
+
+lab.activate(db_prefix + 'lab')
+
+subject.activate(db_prefix + 'subject', linking_module=__name__)
+
+Experimenter = lab.User
+session.activate(db_prefix + 'session', linking_module=__name__)
+
+# Activate "behavior" schema --------------------------------
+
+trial.activate(db_prefix + 'trial', linking_module=__name__)
+# or event? both?
```

Comment:
```suggestion
event.activate(db_prefix + 'event', linking_module=__name__)
```

I would suggest providing examples for both schemas in this `workflow`.